src/documentation/pages/components/sidenav/examples/_base_lumen_centurylink.vue

```diff
@@ -0,0 +1,513 @@
+        .chi-sidenav__drawers
+          .chi-drawer.-animated.-left(v-for="val1 in menuItems" :id="`drawer-${example}-${val1}`")
+            .chi-drawer__header
+              .chi-drawer__title Menu item {{val1}}
+              button(class='chi-button -icon -close' aria-label='Close')
+                .chi-button__content
+                  i.chi-icon.icon-x(aria-hidden="true")
+            .chi-drawer__content
+              .chi-accordion
+                .chi-accordion__item(v-for="val2 in drawerMenuItems" :class="val1 === activeMenuItem && val2 === activeDrawerMenuItem ? '-active -expanded' : ''")
+                  button.chi-accordion__trigger
+                    .chi-accordion__title Item {{val2}}
+                    i.chi-icon.icon-chevron-down(aria-hidden="true")
+                  .chi-accordion__content
+                    .chi-accordion
+                      .chi-accordion__item(v-for="val3 in drawerMenuItems" :class="`${val1 === activeMenuItem && val2 === activeDrawerMenuItem && val3 === activeDrawerMenuItem ? '-active -expanded' : ''}`")
+                        button.chi-accordion__trigger
+                          .chi-accordion__title Item {{val2}}.{{val3}}
+                          i.chi-icon.icon-chevron-down(aria-hidden="true")
+                        .chi-accordion__content
+                          a(v-for="val4 in drawerMenuItems" href=`#exampleHashTarget` :class="`${val1 === activeMenuItem && val2 === activeDrawerMenuItem && val3 === activeDrawerMenuItem && val4 === activeDrawerMenuItem ? '-active' : ''}`")
+                            span Item {{val2}}.{{val3}}.{{val4}}
+                a(href=`#exampleHashTarget`)
+                  span
+                    | External
+                    i.chi-icon.icon-external-link.-xs(aria-hidden="true")
+      .-flex--grow1.-p--3.-pb--4
+        p(v-for="itm in [1, 2, 3, 4]") Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+          | Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. 
+          | Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. 
+          | Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+    <pre class="language-html" slot="code-webcomponent">
+      <code v-highlight="$data.codeSnippets.webcomponent" class="html"></code>
+    </pre>
+    <Wrapper slot="code-htmlblueprint">
+      <JSNeeded />
+      <pre class="language-html">
+        <code v-highlight="$data.codeSnippets.htmlblueprint" class="html"></code>
+      </pre>
+    </Wrapper>
+  </ComponentExample>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+
+declare const chi: any;
+
+@Component({
+  data: () => {
+    return {
+      example: 0,
```

Comment:
if this number never changes i'd suggest just using 0 instead of making it a variable, for better readability above since the connection between "example" and a 0 number is hard to make